examples/example.go

```diff
@@ -0,0 +1,182 @@
+package examples
+
+import (
+	"crypto/tls"
+	"fmt"
+	"math/rand"
+	"net/http"
+	"time"
+
+	"github.com/comvex-jp/uipath-go"
+)
+
+func Run() {
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+
+	credentials := uipath.Credentials{
+		ClientID:   "{{ClientID}}",
+		UserKey:    "{{UserKey}}",
+		TenantName: "{{TenantName}}",
+	}
+
+	httpClient := &http.Client{Transport: tr}
+
+	c := uipath.Client{
+		HttpClient:  httpClient,
+		Credentials: credentials,
+		URLEndpoint: "{{URLEndpoint}}",
+	}
+
+	res, err := uipath.GetOAuthToken(&c)
+	if err != nil {
+		fmt.Println(res, err)
+	}
+
+	c.Credentials.Token = res.AccessToken
```

Comment:
Can we handle the accessToken generation inside the package? Similar implementation we do in the [digima-backend-client`](https://github.com/comvex-jp/digima-backend-client-go)?